bpf/gotracer/go_sql.c

```diff
@@ -27,6 +27,14 @@
 #include <gotracer/go_common.h>
 #include <gotracer/go_str.h>
 
+// Store hostname by goroutine for PostgreSQL (lib/pq)
```

Comment:
Can we please move this code into a standalone file in the maps folder under the go tracer, like we've started doing for other maps?